rocky/assets/js/sidemenuOl.js

```diff
@@ -70,3 +74,71 @@ function addToggleButton(sidemenu) {
 
   prependNode(sidemenu, button);
 }
+
+/**
+ * @param {HTMLElement} sidemenu
+ */
+
+function adjustMaxHeight(sidemenu) {
+  let pageHeight = document.documentElement.scrollHeight;
+
+  const pageHeaderElement = document.getElementById("page-header");
+  const pageFooterElement = document.getElementById("page-footer");
+  const stickyElement = document.getElementById("sticky-container");
+
+  let pageHeaderHeight = pageHeaderElement?.offsetHeight;
+  let pageFooterHeight = pageFooterElement?.offsetHeight;
+  determineSidebarMaxHeight();
+
+  window.addEventListener("resize", (event) => {
+    console.log("resize event");
+    // Set the height of the page header and footer
+    // when the size of the window changes,
+    // to account for responsive behaviour
+    pageHeaderHeight = pageHeaderElement?.offsetHeight;
+    pageFooterHeight = pageFooterElement?.offsetHeight;
+    pageHeight = document.documentElement.scrollHeight;
+    determineSidebarMaxHeight();
+  });
+
+  window.addEventListener("scroll", (event) => {
```

Comment:
Is there no way to do this sticky behavior by using css instead?

Reply:
Not as far as I could find. The combination of variable header and footer heights and multiple sticky elements required some information that css couldn't deliver as far as I could tell. I have discussed this with @underdarknl as well and we've decided to move forward with this pr as is for now. And look into some new css features and possible js fixes that can improve the performance. Which we will add as a separate pr. 

New ticket can be found here: https://github.com/minvws/nl-kat-coordination/issues/3168